src/config.ts

```diff
@@ -0,0 +1,71 @@
+import * as env from "env-var";
+import * as dotenv from "dotenv";
+
+const dotenvPath = process.env.LOAD_DEV_DOTENV ? ".env.dev" : ".env";
+
+dotenv.config({ path: dotenvPath });
+
+const esHost = env
```

Comment:
document all of the environment variables in a file (maybe even in the service)

Reply:
Can you elaborate your review?

Reply:
Write a list of all of the envars with examples and a short explanation so it will be easy to insert to prod. for example:
`NEW_ENV_VARIABLE="example value" // This envar is responsible for x,y,z`.
You can put the list in the description of this PR